js_02/ajax.js

```diff
@@ -0,0 +1,32 @@
+function send(){
+  const url = "./ajax.php";

```

Comment:
Nice job for using relative url. This way if you move some folders around the behavior will be preserved.